src/objectid.ts

```diff
@@ -189,12 +185,12 @@ export class ObjectId {
 
   /**
    * Converts the id into a 24 character hex string for printing
-   *
-   * @param format - The Buffer toString format parameter.
+   * TODO: THIS IS A BREAKAGE.. But implementing this in web is overkill
+   * @param format - hex or base64
    */
-  toString(format?: string): string {
+  toString(format?: 'hex' | 'base64'): string {
```

Comment:
Here again we should not support every format nodejs supported, work around is to get the buffer and call toString directly, continuing to support hex and base64 are common and easy since we have that logic in-house anyway